kotlinpoet-elementhandler-reflective/src/main/kotlin/com/squareup/kotlinpoet/elementhandler/reflective/ReflectiveElementHandler.kt

```diff
@@ -186,6 +186,27 @@ class ReflectiveElementHandler private constructor() : ElementHandler {
     }
   }
 
+  override fun parameterAnnotations(
+    classJvmName: String,
+    methodSignature: JvmMethodSignature,
+    index: Int,
+    isConstructor: Boolean
+  ): List<AnnotationSpec> {
+    val parameters = if (isConstructor) {
+      lookupConstructor(classJvmName, methodSignature)!!.parameters
+    } else {
+      lookupMethod(classJvmName, methodSignature)!!.parameters
+    }
+    return try {
+      parameters[index]
+          .declaredAnnotations
+          .map { AnnotationSpec.get(it, true) }
```

Comment:
nit: Make `true` a named arg for better readability.

Reply:
85ae0a5